test/DynamicExpresso.UnitTest/DynamicTest.cs

```diff
@@ -242,6 +242,26 @@ public void Math_with_dynamic_properties()
 			Assert.AreEqual(500 % dyn.Foo, interpreter.Eval("500 % dyn.Foo"));
 		}
 
+		public class DynamicProperties
+        {
+			public dynamic Foo => 100;
+			public dynamic Bar() => 100;
+        }
+
+		[Test]
+		public void Math_with_dynamic_properties_of_class()
+		{
+
+			var dynamicProperties = new DynamicProperties();
+
+			var interpreter = new Interpreter();
+
+			var del = interpreter.ParseAsDelegate<Func<DynamicProperties, int>>("d.Foo+d.Bar()", new[] { "d" });
+			var r = del.Invoke(dynamicProperties);
+
```

Comment:
Can you just add an assert to verify that the result is the one that we expect?
Can you also check the what happen if you just call `Eval`? What is the return type?
My concern is that in that case the return type will be object? Not that this is a real problem, but just to understand ...

Reply:
Sorry for the missing Assert. I've added tests for the conditions you mentioned. 
The return type of eval is Int32 as expected. The dynamic binding takes care of it as it should.